puts/vendor/scripts/openssl/CMakeLists.txt

```diff
@@ -0,0 +1,167 @@
+cmake_minimum_required(VERSION 3.15)
+
+list(PREPEND CMAKE_MODULE_PATH "${CMAKE_CURRENT_LIST_DIR}/../../../cmake")
+
+if(NOT DEFINED ENV{CC})
+    set(ENV{CC} "clang")
+endif()
+set(CC $ENV{CC})
+
+project(openssl LANGUAGES C)
+
+if(CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
+  # NOTE avoid interactions with system libraries
+  #
+  # The default value of CMAKE_INSTALL_PREFIX lies outside of the project
+  # directory (e.g. /usr/local on UNIX platforms). This prefix is usually part
+  # of the system-wide configuration and might break the system libraries or
+  # their downstream dependencies.
+  #
+  # To be on the safe side, if no prefix is explicitly provided by the caller,
+  # we install the library in an isolated prefix.
+  set(CMAKE_INSTALL_PREFIX "${CMAKE_BINARY_DIR}/install" CACHE PATH "installation directory" FORCE)
+endif()
+
+option(PARALLEL "Activate parallel build" OFF)
+option(WITH_ASAN "Build with address-sanitizer" OFF)
+option(WITH_SANCOV "Build with sancov" OFF)
+option(WITH_GCOV "Build with instrumentation for gcov coverage" OFF)
+option(WITH_LLVM_COV "Build with instrumentation for llvm coverage" OFF)
```

Comment:
Wher would these options be set?

Reply:
This script is the interface between `mk_vendor` and the actual native build system for the vendor library (e.g. configure + make in the case of OpenSSL). The options themselves are defined in `puts/vendor/configs/openssl.toml` and set through the more convenient interface provided by `mk_vendor`.

So you would do:
```sh
$ mk_vendor make openssl:openssl111j --options=gcov # -> pass WITH_GCOV=ON to cmake
$ mk_vendor make openssl:openssl111j --options=sancov,asan # -> pass WITH_SANCOV=ON + WITH_ASAN=ON to cmake
```

Does that answer your question?